### DISCUSSION TITLE
Heat Map on Profile not showing contributions

### DISCUSSION THREAD

**Discussion by Burnsedia** [community/community#65627] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I don't see any feedback on projects at all. I have set my heat map to show private repos as well as public, but I don't get any heat map activity

**Comment by alidehkhodaei** [comment]
Please make sure you have configured your username and email correctly. Use the following commands and then commit again to see the result:
1. `git config --global user.username "Your username github"`
2. `git config --global user.email "Your email"`

The email is the same as the one you used for registering on GitHub.

**Comment by ObsidianBlaze** [comment]
Just for anyone having similar issues, I faced the same recently.
I was contributing to a project but was working on a branch.
The heatmap only got updated when my branch was merged into the default branch, in my case, the `main` branch.
So I belive this should be solved after a merge has been done.
I hope this helps.
